convert.py

```diff
@@ -210,14 +263,67 @@ def convert_text_chunks_to_speech(ebook_chunks):
     print("All chunks converted.")
     return chunk_mp3_filepaths
 
+def merge_wavs_to_mp3(out_filename, path=".", ms_delay=100):
+    combined_mp3 = AudioSegment.empty()
+    path = os.path.abspath(path)
+    print("Joining audio for {}".format(out_filename))
+    for filename in tqdm(sorted(os.listdir(path))):
+        # print("filename: {}, abs: {}, path: {}".format(filename, os.path.abspath(filename), path))

```

Comment:
Can we remove the commented out code? :)

Reply:
Yes - bad habit of mine, I'l; clean out the other files too